.github/workflows/auto-label-pr.yml

```
@@ -0,0 +1,65 @@
+name: Auto Label PR
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)